themes/bootstrap3/js/common.js

```diff
@@ -489,19 +489,6 @@ $(document).ready(function commonDocReady() {
   var url = window.location.href;
   if (url.indexOf('?print=') !== -1 || url.indexOf('&print=') !== -1) {
```

Comment:
Is there a compelling reason to leave some print-related functionality behind in common.js?

Reply:
Without adding a specific setting for it, triggering the print dialog is the only functionality that should be in `trigger_print.js`.

In our use case we still want to show what the printed version of the page looks like, which is what the code left in `common.js` does.

I originally thought of adding a setting, but we already have so many settings, and for many JS functionalities this kind of opt out of/opt in to adding JS files could be a simpler alternative, especially when [VUFIND-1507](https://vufind.org/jira/browse/VUFIND-1507) lands.

Reply:
That makes sense. I'm going to leave this open until @crhallberg has had a chance to weigh in, but I think I'm comfortable merging this if he is.